src/Perlang.Tests.Integration/Typing/TypingTests.cs

```diff
@@ -394,9 +413,11 @@ public void function_return_type_can_specify_explicit_type()
             Assert.Equal("typed return value", output);
         }
 
-        [Fact]
+        [SkippableFact]
         public void function_return_value_can_be_assigned_to_variable_of_same_type()
         {
+            Skip.If(PerlangMode.ExperimentalCompilation, "Not supported in compiled mode");
+
```

Comment:
These exclusions are sad, but is because (non-pointer) `AsciiString` cannot be assigned to `String` (because of C++ copy semantics). I have an idea on how to workaround it though, with a bit of creative hacking... Will give it a try.

Reply:
https://github.com/perlang-org/perlang/pull/453